### ISSUE TITLE
A few language issues

### ISSUE THREAD

**Issue by leschek** [satanasov/phpbbgallery#24]
missing language variables:
On the main page (top right) - NEWS_FILTER_BY_CATEGORY
In ACP - Configure gallery - phpBB integration - RRC_PROFILE_ITEMS

Other stuff:
Small plural issue in statistics (on main page of gallery) where I have written -  "1 images" (should be "1 image")

**Comment by satanasov** [comment]
NEWS_FILTER_BY_CATEGORY - this is not in the gallery, or I can not see it.

'RRC_PROFILE_ITEMS' - added.

**Comment by leschek** [comment]
NEWS_FILTER_BY_CATEGORY - On the main gallery page (top right) it appear when you hover over small triangle under which is menu with items as: View recent images, View Random Images, search etc.

**Comment by satanasov** [comment]
Now I see it ... I will fix it in few hours ...

**Comment by leschek** [comment]
Thank you

**Comment by satanasov** [comment]
Fixed

**Comment by leschek** [comment]
Thank you for fixing it. I don't want to open it, but there is still the small plural issue in statistics (on main page of gallery) where I have written - "1 images" (should be "1 image"). Now I uploaded another image and it shows wrong plural again.

Also, when I'm on main gallery page in albums are informations about number of pictures. I translated the gallery and I think I didn't forget any language line, but word "Images:" is for some reason not translated. I replaced my translation with bg language files and word "Images:" is still shown in English.
